### DISCUSSION TITLE
add links to repos for Core, Supervisor and Host (OS)

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#8690] [Supervisor]
Since this is already in Supervisor mode, and not for the casual HA user, please consider adding the direct links to the 3 repo's just below the headers of the 3 panels.

<img width="980" alt="Schermafbeelding 2021-03-22 om 14 02 29" src="https://user-images.githubusercontent.com/33354141/111993968-8530ab00-8b17-11eb-80c8-784b7d77afbf.png">

this would allow fore quick checks if a ny issues have been written yet before updating, and of course for other related last minute info in the versions of these HA repo's

**Comment by frenck** [comment]
An end-user should not have to interact with the repositories in normal circumstances.

**Reply by Mariusthvdb** [reply]
a well, there are end-users and end-users..... you could say the same for changing the ip-address, or hostname. But also consider subscribing the beta channel. Those end-users most definitely like to keep in touch with latest issues before updating.

<img width="979" alt="Schermafbeelding 2021-03-22 om 14 14 21" src="https://user-images.githubusercontent.com/33354141/111995169-ec029400-8b18-11eb-9a8e-7921f62260e8.png">

more-over the OS updates have been causing several true issues lately upon updating, and the cautious end-user would want to check before clicking Update. 

Added to that, these panels provide info of interest to the more technically inclined end-user. Consider it an 'advanced' feature?

**Reply by frenck** [reply]
I see where you are coming from, but these things are part of the update notification cards/popups. I really think they should not be added to this screen by default.

**Reply by Mariusthvdb** [reply]
I feel for the 'not by default'. (though I wouldn't see any harm in it either, a mere link without any risk of ruining the instance ;-) 
)

That's why I already opted for the 'advanced' mode which is available in HA as it is now. Btw, that is a fine example of a link taking the user to some more explanation what the actual setting is for 

<img width="603" alt="Schermafbeelding 2021-03-22 om 16 18 14" src="https://user-images.githubusercontent.com/33354141/112013277-38a29b00-8b2a-11eb-89bf-70d8a38dba11.png">

My thoughts were to add something like:

**Supervisor**

See [Supervisor](https://github.com/home-assistant/supervisor) for latest developments.

**Reply by ludeeus** [reply]
I do not think this should be added the only users that want this, should subscribe to the repositories or use the GitHub integration to stay up to date.
